clients/ethereumjs/Dockerfile

```
@@ -1,23 +1,31 @@
+### Build Ethereumjs From Git:
+
 FROM node:16-alpine as build
-RUN apk update && apk add --no-cache bash git jq curl python3 gcc make g++ && rm -rf /var/cache/apk/*
-RUN git clone --depth 1 -b develop-v7 https://github.com/ethereumjs/ethereumjs-monorepo.git && \
-    (cd ethereumjs-monorepo && npm i)
 
+ARG user=ethereumjs
+ARG repo=ethereumjs-monorepo
+ARG branch=develop-v7
```

```suggestion
ARG branch=master
```
since develop-v7 is now merged into master